docs/css/custom.css

```diff
@@ -1111,5 +1111,18 @@ img {
 table {
   width: 100%;
   overflow-wrap: normal;
+  border-collapse: collapse; /* Ensures that the borders collapse into a single border */
 }
 
+table th, table td {
+  border: 1px solid #cccccc; /* Adds a border to each table header and data cell */
```

Comment:
I asked ChatGPT to recover the CSS with doc. It looks good so I keep it.

Reply:
The result of the changes are done by myself.

Reply:
why dropping the striped this is exactly what the bootstrap theme already provides us. 

The easier way would be simply to 

```
.table {
  border-collapse: collapse;
  border: 1px solid #ccc
}
```

Reply:
I tried the striped style but it was not very clear when there is long text ( you can find it in the commit history):
<img width="918" alt="image" src="https://github.com/apache/spark/assets/1097932/8e1eca17-a44d-43ca-9b58-0f96f1995e2d">